apps/backend/src/modules/courses/course.controller.ts

```diff
@@ -132,6 +139,37 @@ class CourseController extends BaseController {
 				params: courseIdParameterValidationSchema,
 			},
 		});
+		this.addRoute({
+			handler: () => {
+				return this.findAll();
+			},
+			method: "GET",
+			path: CoursesApiPath.ALL,
+			preHandler: checkUserPermissions(
+				[PermissionKey.MANAGE_COURSES],
+				PermissionMode.ALL_OF,
+			),
+		});
+		this.addRoute({
+			handler: (options) => {
+				return this.updateByAdmin(
+					options as APIHandlerOptions<{
+						body: CourseUpdateRequestDto;
+						params: { courseId: string };
+					}>,
+				);
+			},
+			method: "PATCH",
```

Comment:
```suggestion
			method: "PUT",
```
let's implement put (update)

Reply:
We already have put on `courses/:courseId` (update course from vendor). So there is conflict with it.

Reply:
courses/:courseId - should update the course

Reply:
courses/:courseId - should update the course